src/System Application/App/SFTP Client/src/DotnetSFTPClient.Codeunit.al

```diff
@@ -0,0 +1,239 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.SFTPClient;
+
+using System;
+
+codeunit 9760 "Dotnet SFTP Client" implements "ISFTP Client"
+{
+    Access = Internal;
+    InherentEntitlements = X;
+    InherentPermissions = X;
+
+    var
+        RenciSFTPClient: DotNet RenciSftpClient;
+        LastOperationSuccessful: Boolean;
+
+    procedure Disconnect()
+    begin
+        RenciSFTPClient.Disconnect();
+    end;
+
+    procedure IsConnected(): Boolean
+    begin
+        exit(RenciSFTPClient.IsConnected());
+    end;
+
+    procedure Exists(Path: Text; var ExistResult: Boolean): Boolean
+    begin
+        LastOperationSuccessful := InternalExists(Path, ExistResult);
+        exit(LastOperationSuccessful);
+    end;
+
+    [TryFunction]
+    local procedure InternalExists(Path: Text; var ExistResult: Boolean)
```

Comment:
nit: I personally don't like these names `Internal<functionality>`, `Try<functionality>` is more explicit in its functionality. `TryExists` in this case.